packages/ZbayLite/src/rtl-tests/community.join.test.tsx

```diff
@@ -51,6 +52,14 @@ describe('User', () => {
 
     const factory = await getFactory(store)
 
+    const quietCommunity = await factory.create<
+      ReturnType<typeof communities.actions.addNewCommunity>['payload']
+    >('Community')
+
+    const quietUser = await factory.create<
```

Comment:
This is the same thing as above but also there's no need for creating additional user as we already have one being created at line 69 (you should only add test-scope accessible reference)